### ISSUE TITLE
Invalid jsons in db

### ISSUE THREAD

**Issue by florczakraf** [florczakraf/stepmania-chart-db#2]
db/fd/ce5ae1652dd556.json: Extra data: line 1 column 280 (char 279)

```
{"subtitle":"","hash":"fdce5ae1652dd556","title":"Birthday","titletranslit":"","pack_name":"Stomp Exceed","steps_type":"dance-double","artist":"The Cruxshadows","diff":"Easy","diff_number":2,"subtitletranslit":"","bpms":"0.000=132.000","artisttranslit":"","directory":"birthday"}"The Cruxshadows"}
```
there might be more than just this one. We need to figure out if it's a result of manual modification or yet another problem with the lua json encoder.

**Comment by florczakraf** [comment]
```
db/fd/ce5ae1652dd556.json
{"subtitle":"","hash":"fdce5ae1652dd556","title":"Birthday","titletranslit":"","pack_name":"Stomp Exceed","steps_type":"dance-double","artist":"The Cruxshadows","diff":"Easy","diff_number":2,"subtitletranslit":"","bpms":"0.000=132.000","artisttranslit":"","directory":"birthday"}"The Cruxshadows"}

db/63/5bbf0e8c59feae.json
{"artist":"SoundTeMP","artisttranslit":"","bpms":"0.000=150.002","diff_number":2,"subtitletranslit":"","titletranslit":"","title":"Monastery In Disguise","pack_name":"Stomp Exceed","directory":"monastery in disguise","hash":"635bbf0e8c59feae","diff":"Easy","steps_type":"dance-double","subtitle":"RO BGM #113"}2","artist":"SoundTeMP"}

db/48/6f984e6056e17a.json
{"subtitle":"","diff_number":10,"subtitletranslit":"","diff":"Challenge","artisttranslit":"","artist":"Beltaine","titletranslit":"","bpms":"0.000=150.000,127.000=132.000,127.250=120.000,127.500=112.000,127.750=104.000,128.000=96.000,128.250=88.000,128.500=80.000,129.000=78.000,130.000=76.500,131.000=75.000,188.000=72.000,189.000=67.000,190.000=62.000,191.000=47.000,192.000=57.400,193.000=55.500,194.000=51.500,195.000=150.000","hash":"486f984e6056e17a","title":"Foggy Dew","pack_name":"Stomp Exceed","directory":"Foggy Dew","steps_type":"dance-double"}btitle":""}

db/3c/8225f6f7f650c1.json
{"diff_number":2,"titletranslit":"","artist":"F-777","subtitle":"","hash":"3c8225f6f7f650c1","subtitletranslit":"","diff":"Beginner","title":"Come On","steps_type":"dance-single","pack_name":"Rikame's Simfiles 3","directory":"Come On","artisttranslit":"","bpms":"0.000=140.000"}tle":""}

db/89/b5165f1c95f658.json
{"diff_number":9,"titletranslit":"","artist":"F-777","subtitle":"","hash":"89b5165f1c95f658","subtitletranslit":"","diff":"Hard","title":"Come On","steps_type":"dance-single","pack_name":"Rikame's Simfiles 3","directory":"Come On","artisttranslit":"","bpms":"0.000=140.000"}tle":""}

db/62/16ff81b0070331.json
{"artist":"ripple","diff_number":5,"titletranslit":"","steps_type":"dance-single","subtitle":"","artisttranslit":"","pack_name":"Rikame's Simfiles 4","directory":"Undermine","bpms":"0.000=135.938,87.000=229.000,88.000=135.938,102.750=298.000,103.500=135.938,184.000=67.995,191.500=248.000,192.000=67.995,202.000=136.000","diff":"Easy","subtitletranslit":"","hash":"6216ff81b0070331","title":"Undermine"}itle":""}

db/92/8cb4fa7f18db24.json
{"artisttranslit":"","subtitletranslit":"","subtitle":"","hash":"928cb4fa7f18db24","titletranslit":"","title":"T3","artist":"2M","bpms":"0.000=156.000","pack_name":"Rikame's Simfiles 2","directory":"T3","diff_number":11,"diff":"Challenge","steps_type":"dance-single"}b24"}

db/a7/44e28435e82ee5.json
{"subtitle":"Crash Bash","hash":"a744e28435e82ee5","pack_name":"SimFile.pl","bpms":"0.000=156.110","diff_number":6,"titletranslit":"","steps_type":"dance-single","artisttranslit":"","diff":"Medium","directory":"Bearminator - Steve Duckworth [Greg]","artist":"Steve Duckworth","subtitletranslit":"","title":"Bearminator"}anslit":""}

db/cb/d196b93b9a8b9f.json
{"steps_type":"dance-double","title":"Labour Of Love","artisttranslit":"","diff":"Medium","subtitle":"","directory":"labor of love","subtitletranslit":"","bpms":"0.000=125.000,201.000=120.000,201.250=110.000,201.500=100.000,201.750=90.000,202.000=80.000,202.250=70.000,202.500=60.000,202.750=50.000,202.875=45.000,203.000=40.000,203.125=35.000,203.250=30.000,203.375=25.000,203.500=20.000,203.625=15.000,203.750=10.000,203.875=5.000","titletranslit":"","pack_name":"Stomp Exceed","diff_number":5,"artist":"Dead Can Dance","hash":"cbd196b93b9a8b9f"}","diff":"Medium"}

db/e2/b3a6a4c74cdc9d.json
{"bpms":"0.000=127.000,55.250=190.500,56.000=127.000,208.000=190.500,208.750=63.500,209.000=127.000,224.000=190.500,224.750=63.500,225.000=127.000,238.000=190.500,238.750=63.500,239.000=127.000","artisttranslit":"","diff_number":9,"titletranslit":"","artist":"Fusq","diff":"Challenge","directory":"[09] Perfume!","hash":"e2b3a6a4c74cdc9d","pack_name":"ITG Eurocup 2018 - Precision","steps_type":"dance-single","subtitle":"","title":"[09] Perfume!","subtitletranslit":""}fume!"}

db/2f/8c71e9b51d2654.json
{"subtitle":"Crash Bash","hash":"2f8c71e9b51d2654","pack_name":"SimFile.pl","bpms":"0.000=156.110","diff_number":3,"titletranslit":"","steps_type":"dance-single","artisttranslit":"","diff":"Easy","directory":"Bearminator - Steve Duckworth [Greg]","artist":"Steve Duckworth","subtitletranslit":"","title":"Bearminator"}anslit":""}
```

**Comment by florczakraf** [comment]
looks like the issue has been already solved since 4e90b6b50061ae5fd56a6d349d863483d2710daf
